From 4c2ffa56713d610d64c4f3ce653402fa663ea5a7 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Sun, 29 Mar 2020 17:50:48 +0200
Subject: [PATCH 1/2] remove pyproject.toml

---
 pyproject.toml | 2 --
 1 file changed, 2 deletions(-)
 delete mode 100644 pyproject.toml

diff --git a/pyproject.toml b/pyproject.toml
deleted file mode 100644
index 21722a56..00000000
--- a/pyproject.toml
+++ /dev/null
@@ -1,2 +0,0 @@
-[build-system]
-requires = ["numpy >= 1.15", "scipy >= 1.1.0", "setuptools>=40.8.0", "wheel"]
\ No newline at end of file
-- 
2.29.2.windows.3

