From 211c4f53a5fbf79ee989a1e337bf57e4456cce5e Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Sun, 29 Mar 2020 16:51:28 +0200
Subject: [PATCH] remove requires from setup.py and pyproject.toml

---
 pyproject.toml |  5 +++--
 setup.py       | 15 ++++++++-------
 2 files changed, 11 insertions(+), 9 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 21722a56..ac05e964 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,2 +1,3 @@
-[build-system]
-requires = ["numpy >= 1.15", "scipy >= 1.1.0", "setuptools>=40.8.0", "wheel"]
\ No newline at end of file
+### conda recipe takes care of installing requirements
+#[build-system]
+# requires = ["numpy >= 1.15", "scipy >= 1.1.0", "setuptools>=40.8.0", "wheel"]
diff --git a/setup.py b/setup.py
index 90ed0fc6..328e3bab 100644
--- a/setup.py
+++ b/setup.py
@@ -65,12 +65,13 @@ setup(
     license='Apache License, Version 2.0',
     zip_safe=False,
     description='A domain-specific language for modeling convex optimization problems in Python.',
-    install_requires=["osqp >= 0.4.1",
-                      "ecos >= 2",
-                      "scs >= 1.1.3",
-                      "multiprocess",
-                      "numpy >= 1.15",
-                      "scipy >= 1.1.0"],
-    setup_requires=["numpy >= 1.15"],
+    ### conda recipe takes care of installing requirements
+    # install_requires=["osqp >= 0.4.1",
+    #                   "ecos >= 2",
+    #                   "scs >= 1.1.3",
+    #                   "multiprocess",
+    #                   "numpy >= 1.15",
+    #                   "scipy >= 1.1.0"],
+    # setup_requires=["numpy >= 1.15"],
     use_2to3=True,
 )
-- 
2.24.0.windows.2

