From 7d087d5781a5e9deaf3e4459376d93d5d700e4b0 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Wed, 4 Jan 2023 15:17:01 +1100
Subject: [PATCH 1/2] fix relative import

---
 cvxpy/tests/test_perspective.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/cvxpy/tests/test_perspective.py b/cvxpy/tests/test_perspective.py
index 957c0683b..e7d841008 100644
--- a/cvxpy/tests/test_perspective.py
+++ b/cvxpy/tests/test_perspective.py
@@ -18,10 +18,9 @@ import numpy as np
 import pytest
 
 import cvxpy as cp
+from cvxpy.atoms.perspective import perspective
 from cvxpy.constraints.exponential import ExpCone
 
-from ..atoms.perspective import perspective
-
 
 def test_monotonicity():
     x = cp.Variable(nonneg=True)
-- 
2.38.1.windows.1

