From 0e31bfb02ccb8ec8fd77850fba4dc53b31bf01a6 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Sun, 29 Mar 2020 17:50:48 +0200
Subject: [PATCH 1/2] remove pyproject.toml

---
 pyproject.toml | 19 -------------------
 1 file changed, 19 deletions(-)
 delete mode 100644 pyproject.toml

diff --git a/pyproject.toml b/pyproject.toml
deleted file mode 100644
index 12cd1a9c9..000000000
--- a/pyproject.toml
+++ /dev/null
@@ -1,19 +0,0 @@
-[tool.isort]
-include_trailing_comma = true
-use_parentheses = true
-extend_skip = [
-    "cvxpy/__init__.py",
-    "cvxpy/reductions/__init__.py",
-    "cvxpy/reductions/dcp2cone/atom_canonicalizers/__init__.py"
-]
-
-[build-system]
-requires = [
-    "numpy>=1.15,<1.16; python_version=='3.6'",
-    "numpy>=1.15,<1.16; python_version=='3.7'",
-    "numpy>=1.16,<1.17; python_version=='3.8'",
-    "numpy>=1.19,<1.20; python_version=='3.9'",
-    "scipy >= 1.1.0",
-    "setuptools>=40.8.0",
-    "wheel"
-]
\ No newline at end of file
-- 
2.32.0.windows.2

