From c814b080bba947caf1e76ff1ca1669a21da95953 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Sun, 10 Jan 2021 23:12:39 +0100
Subject: [PATCH 2/2] do not use builtins as a dict

---
 cvxpy/tests/test_problem.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/cvxpy/tests/test_problem.py b/cvxpy/tests/test_problem.py
index 5a1ba1db..581c1a77 100644
--- a/cvxpy/tests/test_problem.py
+++ b/cvxpy/tests/test_problem.py
@@ -31,6 +31,7 @@ from numpy import linalg as LA
 import numpy
 import numpy as np
 import scipy.sparse as sp
+import builtins
 import sys
 import pickle
 # Solvers.
@@ -1693,7 +1694,7 @@ class TestProblem(BaseTest):
         prob = Problem(cp.Minimize(cp.power(x, 1.7) + cp.power(x, -2.3) - cp.power(x, .45)))
         prob.solve()
         x = x.value
-        self.assertTrue(__builtins__['abs'](1.7*x**.7 - 2.3*x**-3.3 - .45*x**-.55) <= 1e-3)
+        self.assertTrue(builtins.abs(1.7*x**.7 - 2.3*x**-3.3 - .45*x**-.55) <= 1e-3)
 
     def test_multiply_by_scalar(self):
         """Test a problem with multiply by a scalar.
-- 
2.29.2.windows.3

