From 9ba07a28ef8b61fe6e6351931d7fa82e3067482d Mon Sep 17 00:00:00 2001
From: phschiele <phschiele@gmail.com>
Date: Wed, 10 Nov 2021 14:35:19 +0100
Subject: [PATCH 2/2] Fix test

---
 cvxpy/tests/test_atoms.py | 12 ++++--------
 1 file changed, 4 insertions(+), 8 deletions(-)

diff --git a/cvxpy/tests/test_atoms.py b/cvxpy/tests/test_atoms.py
index db98a1df9..2ba6d24db 100644
--- a/cvxpy/tests/test_atoms.py
+++ b/cvxpy/tests/test_atoms.py
@@ -17,6 +17,7 @@ limitations under the License.
 import unittest
 
 import numpy as np
+import pytest
 import scipy
 import scipy.sparse as sp
 import scipy.stats
@@ -293,17 +294,12 @@ class TestAtoms(BaseTest):
     def test_elemwise_arg_count(self) -> None:
         """Test arg count for max and min variants.
         """
-        with self.assertRaises(Exception) as cm:
+        error_message = r"__init__\(\) missing 1 required positional argument: 'arg2'"
+        with pytest.raises(TypeError, match=error_message):
             cp.maximum(1)
-        self.assertTrue(str(cm.exception) in (
-            "__init__() takes at least 3 arguments (2 given)",
-            "__init__() missing 1 required positional argument: 'arg2'"))
 
-        with self.assertRaises(Exception) as cm:
+        with pytest.raises(TypeError, match=error_message):
             cp.minimum(1)
-        self.assertTrue(str(cm.exception) in (
-            "__init__() takes at least 3 arguments (2 given)",
-            "__init__() missing 1 required positional argument: 'arg2'"))
 
     def test_matrix_frac(self) -> None:
         """Test for the matrix_frac atom.
-- 
2.32.0.windows.2

