From 3da934d122066857628d2374f5a272da4a30d602 Mon Sep 17 00:00:00 2001
From: phschiele <44360364+phschiele@users.noreply.github.com>
Date: Sat, 5 Nov 2022 16:13:28 -0700
Subject: [PATCH 2/2] disallow float for diamatrix values (#1931)

---
 cvxpy/atoms/elementwise/elementwise.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cvxpy/atoms/elementwise/elementwise.py b/cvxpy/atoms/elementwise/elementwise.py
index f02164efb..d59dcc289 100644
--- a/cvxpy/atoms/elementwise/elementwise.py
+++ b/cvxpy/atoms/elementwise/elementwise.py
@@ -60,7 +60,7 @@ class Elementwise(Atom):
         """
         if not np.isscalar(value):
             value = value.ravel(order='F')
-        return sp.dia_matrix((value, [0]), shape=(rows, cols)).tocsc()
+        return sp.dia_matrix((np.atleast_1d(value), [0]), shape=(rows, cols)).tocsc()
 
     @staticmethod
     def _promote(arg, shape: Tuple[int, ...]):
-- 
2.38.1.windows.1

